docker/conf/docker-compose-prod-deploy.yml.dist

```diff
@@ -0,0 +1,63 @@
+version: "3.1"
+services:
+    webserver:
+        restart: always
+        image: nginx:1.13-alpine
+        container_name: production-webserver
+        volumes:
+            - /var/www/production-content:/var/www/html/web/content
+            - ./web:/var/www/html/web
+            - nginx-conf:/etc/nginx/conf.d/
+        ports:
+            - 8000:8080
+        expose:
+            - 8000
+        links:
+            - php-fpm
+        networks:
+            - shopsys-network
+
+    php-fpm:
+        restart: always
+        image: production-php-fpm
+        container_name: production-php-fpm
+        volumes:
+            - /var/www/production-content:/var/www/html/web/content
```

Comment:
What is production-content?

Reply:
shared folder where the application stores files uploaded from administration that should not change. This should be the target folder for flysystem.